_data/toc.yaml

```diff
@@ -464,78 +464,10 @@ guides:
   - path: /compliance/cis/
     title: CIS
 
-- sectiontitle: Open source at Docker
+- sectiontitle: Contributors guide
```

Comment:
Should we call it something like Contribute to the docs? So it's parallel with the other top level things? Or maybe it doesn't need its own top level section anymore?